src/testrender/cuda/rend_lib.h

```diff
@@ -6,11 +6,15 @@
 
 #include <OSL/oslconfig.h>
 
-#include <Imath/half.h>
-
 #include <OSL/hashes.h>
 #include <OSL/oslexec.h>
 
+#include "../raytracer.h"
+
+
+#define RAYTRACER_HIT_QUAD   0
```

Comment:
Leftover from pre-triangle version?

Reply:
Yeah, it must have snuck back in during my manual rebase.